recipes/selectolax/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "selectolax" %}
+{% set version = "0.3.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/selectolax-{{ version }}.tar.gz
+  sha256: d71d7ae7bec6e856b83111f270bc783e531d86e02ad3a78a37d8591ff8f3620f
+
+build:
+  number: 0
+  noarch: python
```

Comment:
If you’re using a compiler, it cannot be noarch.

Reply:
Thank you. I was unsure on this part.

Reply:
Removed `noarch`.